docs/storefront/store-management.md

```diff
@@ -6,11 +6,155 @@ Zustand allows us to create multiple stores. Each store consists of `slices`, wh
+```
+
+### Changes to your persist store's schema, versioning, and migrations
+
+It is possible that you will need to make a breaking change to your schema. These could be, for example, one of the scenarios below:
+
+-   a key is added to the schema
+-   a key is removed from the schema
+-   a key is renamed
+-   a value for a given key changes schematically (e.g. was nullable but is not anymore, could be a number but now can only be a string, etc.)
+
+In such situation, these steps will tell you exactly what to do:
+
+#### 1. Increase the version of your store
+
+```diff
+export const usePersistStore = create<PersistStore>()(
+    persist(
+        ...
+        {
+            ...
+-            version: 1,
++            version: 2,
+```
+
+#### 2. Add migration logic
+
+```ts
+export const usePersistStore = create<PersistStore>()(
+    persist(
+        ...,
+        {
+            ...,
+            migrate: (persistedState, version) => {
+                let migratedPersistedState = { ...(persistedState as object) };
+
+                if (version === 2) {
+                    delete migratedPersistedState.oldKey;
+                    migratedPersistedState.newKey = 'newValue';
+
+                    if (migratedPersistedState.changedKey === null) {
+                        migratedPersistedState.changedKey = 0;
+                    } else {
+                        migratedPersistedState.changedKey = 1;
+                    }
+                }
+
+                return migratedPersistedState as PersistStore;
+            },
+        }
+    ),
+);
+```
+
+#### 3. Add a base version below which the store is just reset (optional)
+
+This can be useful if the newest version is e.g. 100, and you only want to migrate the last 10 versions (which could mean visiting the app and using the store sometimes during the last year). If you want to simply reset all stores with versions lower than 90, as they are too old and migrating them would be too complicated, this is how it can be done
+
+```ts
+export const usePersistStore = create<PersistStore>()(
+    persist(
+        ...,
+        {
+            ...,
+            migrate: (persistedState, version) => {
+                let migratedPersistedState = { ...(persistedState as object) };
+
+                if (version < 90) {
+                    migratedPersistedState = {
+                        ...defaultAuthLoadingState,
+                        ...defaultUserState,
+                        ...defaultContactInformationState,
+                        ...defaultPacketeryState,
+                    };
+                }
+                ...
+
+                return migratedPersistedState as PersistStore;
+            },
+        }
+    ),
+);
+```
+
+#### 4. Fix the `DEFAULT_APP_STORE` constant in your cypress (optional)
+
+Remember to update the `DEFAULT_APP_STORE` constant in your cypress so that it mirrors the actual default state set in your persist store.
```

Comment:
Isn't the constant named `DEFAULT_PERSIST_STORE_STATE`? 
I did not find the DEFAULT_APP_STORE constant

Reply:
You are right. Thanks. Refactored.